packages/txservice/src/shared/contracts.ts

```diff
@@ -32,7 +40,8 @@ export const CHAINS_WITH_PRICE_ORACLES: number[] = ((): number[] => {
  * deployed.
  */
 export const getDeployedPriceOracleContract = (chainId: number): { address: string; abi: any } | undefined => {
-  const record = (contractDeployments as any)[String(chainId)] ?? {};
+  const _contractDeployments = getContractDeployments();
+  const record = _contractDeployments[String(chainId)] ?? {};
```

Comment:
I know you didn't write this code, but this should use `chainId.toString()`, not `String(chainId)`

Reply:
yeah, better practice to use toString(). We were doing this at multiple files, updated all.